.cache/eleventy-fetch-11315cd0bbd2007d85b0862d9403b3

```diff
@@ -0,0 +1 @@
+{"cachedAt":1755546441849,"type":"buffer","metadata":{"response":{"url":"https://designsystem.digital.gov/img/introducing-uswds-2-0/built-to-grow--alt.jpg","status":202,"headers":{"access-control-allow-methods":"OPTIONS,GET,POST","access-control-allow-origin":"*","access-control-expose-headers":"x-amzn-waf-action","access-control-max-age":"86400","cache-control":"no-store, max-age=0","connection":"keep-alive","content-length":"0","content-type":"text/html; charset=UTF-8","date":"Mon, 18 Aug 2025 19:47:21 GMT","server":"CloudFront","via":"1.1 0286f8e6d2ddfae7e9a56c7cc839488c.cloudfront.net (CloudFront)","x-amz-cf-id":"E_fl6HMgNnS5RHvgJRmDwUsXbTclNObq5HN1i7lDUuHXyz-jJQy7Aw==","x-amz-cf-pop":"IAD12-P2","x-amzn-waf-action":"challenge","x-cache":"Error from cloudfront"}}}}
```

Comment:
We don't want these `.cache` files committed. Could we add `.cache/` to the .gitignore file and remove them from this PR?